styles/header.css

```diff
@@ -0,0 +1,75 @@
+.nav {
+  display: flex;
+  height: 48px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.left-section {
+  margin-left: 12px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+}
+
+.logo {
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  padding: 10px 12px;
+}
+
+.right-section {
+  margin-right: 24px;
+  margin-top: 4px;
+  margin-bottom: 4px;
+  border: none;
+  background-color: white;
+}
+
+.icon-menu {
+  padding: 8px;
+  width: 24px;
+  height: 24px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  margin-left: 24px;
+  margin-right: 24px;
+  background-image: url('/Module-1/My-portfolio/images/header-shapes\ mobile@2x.svg');
```

Comment:
- [ ] Kindly make sure you use the appropriate path for the background image as it is shown in the picture below your background image is not  visible  please fix that and make sure your image is displayed as expected it will make your page look nice and stick to the Figma design as required in the project :+1: 

## Your design 
![image](https://user-images.githubusercontent.com/95297251/205052450-0266c290-8ae3-4205-960b-06293b28753a.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/205052816-5e4f4dbe-a856-466f-ab48-b11913dea805.png)